File: project3/Team3/projectBQ/src/main/java/bitc/full502/projectbq/dto/StockLogDto.java
package bitc.full502.projectbq.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class StockLogDto {

    private long id;
    private String empCode;
    private String empName;
    private LocalDateTime logDate;
    private String type;
    private long quantity;
    private String memo;

    private long warehouseId;
    private String warehouseName;
    private String warehouseKrName;

    private long itemId;
    private String itemCode;
    private String itemName;
    private String itemManufacturer;
    private long itemPrice;

    private long categoryId;
    private String categoryName;
    private String categoryKrName;
}

File: project3/Team3/App_BQ/app/src/main/java/bitc/full502/app_bq/data/model/StockLogDto.kt
package bitc.full502.app_bq.data.model

import java.time.LocalDateTime

data class StockLogDto(
    val id: Long,
    val empCode: String?,
    val empName: String?,
    val logDate: String?,
    val type: String?,
    val quantity: Long,
    val memo: String?,

    val warehouseId: Long,
    val warehouseName: String?,
    val warehouseKrName: String?,

    val itemId: Long,
    val itemCode: String?,
    val itemName: String?,
    val itemManufacturer: String?,
    val itemPrice: Long,

    val categoryId: Long,
    val categoryName: String?,
    val categoryKrName: String?
)
